Homework 1 Phys 250 Robin James rnj13

# Problem 1 #
## Part i ##
The perimeter of a square with sides S is 4S. For the propose problem we can write the sum of N squares as:
$$ 4S + 2S+ S+ \frac{S}{2}+... \frac{4S}{2^{N-1}} = \[ \sum_{i=0}^{N-1} \frac{4S}{2^{i}} \] = 4S(\frac{1-(.5S)^N}{1-.5S})$$
As N -> $ \infty $ the perimeter goes to  $$\frac{4S}{1-\frac{S}{2}} $$
## Part ii ##
The area of the square with sides S is $S^2$. The sum of the area is:
$$ \[ \sum_{i=0}^{N-1} \frac{S^2}{2^{i}} \] $$

# Problem 2 #
Set your coordinate "a" to be parallel to the incline of the plane. In the reference frame, the normal force and perpendicular part of the gravitational force. We only need to use Newton's law in the "a" direction
$$ F_{net{= mgsin( \theta) $$
$$ a_a=gsin(\theta) $$
We can then plug back in to a kinematic equation to get x(t).
$$ d= \frac{1}{2}gsin(\theta)t^2 $$
## Part ii ##
$$ 4= .5 g sin(\theta) => \theta = arcsin(\frac{8}{g})= 0.955 rad  $$
## Part iii ##
$$ 0= \frac{g}{4\omega^2}(e^{\omega}-e^{-\omega}-2 sin(\omega)) -4  $$
To solve this I will use scipy root finder

In [16]:
import numpy as np
import scipy.optimize as opt

def f(w):
    return (9.8/(4.*w**2.))*(np.exp(w)-np.exp(-1*w)-2.*np.sin(w))-4.

Now with the function defined, we need to find two numbers where we think the root is.Since we know $\omega$ is an angular speed in rad/sec so we can make a guess that it will be greater than 0.01 but less than pi.

In [24]:
opt.brentq(f,0.01,3.1,xtol=10e-15)

2.3612521534858644

Running using brentq, we see that $\omega = 2.3612521534858644$rad/s
## Part iv ##
Same proccess as last part.

In [25]:

def f(w):
    return (9.8/(4.*w**2.))*(np.exp(w)-np.exp(-1*w)-2.*np.sin(w))-8.


The speed must be faster then last one but still less than 2pi

In [30]:
opt.brentq(f,2.36,6,xtol=10e-15)

3.855403679362072

To slide 8 meters, $\omega=3.855403679362072$ rad/s

# Problem 3 #